rules.ini

```
@@ -158,6 +158,7 @@ Adobe_Flash[] = (?:^|/)StandalonePlayerLocalizable\.strings(?:$|/)
 AMD_FSR2 = (?:^|/)ffx_fsr2_api_x(?:64|86)\.dll$
 AMD_GPU_Services = (?:^|/)amd_ags_x(?:64|86)\.dll$
 Azure_Playfab_Party = (?:^|/)Party(?:Win|Win7|Win32|XboxLive|XboxLiveWin32)\.dll$
+BASS = (?:^|/)(?:bass\.dll|bassflac\.dll|bassmidi\.dll|libbass\.dylib|libbass\.so)$
```

let's make it less horrible to look at
```suggestion
BASS = (?:^|/)(?:bass(?:flac|midi|)\.dll|libbass\.(?:dylib|so))$
```